experiments/ClimaEarth/run_amip.jl

```diff
@@ -139,7 +139,7 @@ postprocessing and the conservation checks) of the simulation will be saved,
 #and `dir_paths.checkpoints`, where restart files are saved.
 =#
 
-COUPLER_OUTPUT_DIR = joinpath(output_dir_root, job_id)
+COUPLER_OUTPUT_DIR = joinpath(pkg_dir, output_dir_root, job_id)
```

Comment:
I don't think it should not be relative to `pkg_dir`. `pkg_dir` could be inside the `.julia` in the user's home directory. This is a hidden folder, with possibly storage quotas. It is not a great place where to save files.